_posts/en/newsletters/2020-07-15-newsletter.md

```diff
@@ -0,0 +1,102 @@
+  proposals describe optional signature hash (sighash) flags that do not
+  commit to the particular UTXOs (inputs/previous outputs) being spent
+  in a transaction, making it possible to create a signature for a
+  transaction without knowing what UTXO it will spend.  That feature can
+  be used by the proposed [Eltoo][topic eltoo] settlement layer to allow
+  creating a series of transactions where any later transaction can
+  spend the value from any earlier transaction, allowing an offchain
+  contract to be settled in its latest state even if earlier states get
+  confirmed onchain.
+
+    The main difference between the noinput and anyprevout proposals is
+    that noinput would require its own new version of segwit but
+    anyprevout uses one of upgrade features of the proposed [BIP342][]
+    specification of [tapscript][topic tapscript].  Additional
+    differences between the proposals are described in the [revised
+    text][anyprevout revisions] itself.
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+FIXME:none so far; harding to update or delete this section Tuesday
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[C-Lightning][c-lightning repo], [Eclair][eclair repo], [LND][lnd repo],
+[Rust-Lightning][rust-lightning repo], [libsecp256k1][libsecp256k1 repo],
+[Hardware Wallet Interface (HWI)][hwi], [Bitcoin Improvement Proposals
+(BIPs)][bips repo], and [Lightning BOLTs][bolts repo].*
+
+- [Bitcoin Core #19219][] FIXME:jonatack, also [Bitcoin Core #19469][]
+
+- [Bitcoin Core #19328][] updates the `gettxoutsetinfo` RPC with a new
+  `hash_type` parameter that allows specifying how to generate a
+  checksum of the current UTXO set.  Currently the only two options are
+  `hash_serialized_2`, which produces the checksum that has been the
+  default since Bitcoin Core 0.15 (September 2017), or `none`, which
+  returns no checksum.  It's [planned][Bitcoin Core #18000] to later
+  allow a [muhash][]-based checksum along with an index that will allow
+  returning the checksum much more quickly than is now possible.  For
+  now, requesting the `none` result allows the `gettxoutsetinfo` RPC to
+  run much more quickly, which is useful for anyone running it after
+  each block (e.g. to audit the number of spendable bitcoins).  For
+  historical context on fast UTXO set checksums, see this [2017
+  post][wuille rolling] by Pieter Wuille.
+
+- [Bitcoin Core #19191][] updates the `-whitebind` and `-whitelist`
+  configuration settings with a new `download` permission.  When this
+  permission is applied to a peer, it will be allowed to continue
+  downloading from the local node even if the node has reached its
+  `-maxuploadtarget` maximum upload limit.  This makes it easy for a
```

Comment:
perhaps mention the `download` flag is for the moment a subset of the `noban` flag and was extracted to enable finer-grained permissioning

Reply:
This is mentioned in the final sentence of the paragraph, "The existing `noban` permission also gives peers
  with that permission an unlimited download capability, but that may be
  changed in a future release."